### ISSUE TITLE
Che Theia build failed

### ISSUE THREAD

**Issue by mmorhun** [eclipse-che/che#14685]
### Is your task related to a problem? Please describe.
After [some changes ](https://github.com/eclipse-theia/theia/pull/6252)in Theia upstream, build of Che Theia is broken.
We need to make corresponding changes in Che Theia.

**Comment by mmorhun** [comment]
We're temporary switching to `0.11.0` version of Che Theia (see https://github.com/eclipse/che-theia/pull/451), so I think we can lower a bit issue severity.

**Comment by tolusha** [comment]
I personally don't want to postpone for uncertain amount of time, that's why I prefer `blocker`.

@benoitf 
WDYT ?

**Comment by benoitf** [comment]
@tolusha well I will propose another PR which will fix the code. But it unblocks all ppl now.

**Comment by benoitf** [comment]
@tolusha I mean, I will work on it, it's not to push it back until 3 weeks

**Comment by dmytro-ndp** [comment]
More info about build failure: https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Che-Theia-master-check-E2E-Happy-path-tests-against-K8S/125/console
```
lerna ERR! $ yarn clean && yarn build
lerna ERR! $ rimraf lib
lerna ERR! $ concurrently -n "format,lint,compile" -c "red,green,blue" "yarn format" "yarn lint" "yarn compile" && yarn run compileWorker
lerna ERR! [format] $ tsfmt -r --useTsfmt ../../configs/tsfmt.json
lerna ERR! [compile] $ tsc
lerna ERR! [lint] $ tslint -c ../../configs/tslint.json --project tsconfig.json
lerna ERR! [format] yarn format exited with code 0
lerna ERR! [lint] yarn lint exited with code 0
lerna ERR! [compile] src/browser/plugin/che-plugin-frontend-service.ts(39,56): error TS2339: Property 'getDeployedMetadata' does not exist on type 'HostedPluginServer'.
lerna ERR! [compile] src/browser/plugin/che-plugin-frontend-service.ts(42,36): error TS2339: Property 'source' does not exist on type 'PluginMetadata'.
lerna ERR! [compile] src/browser/plugin/che-plugin-frontend-service.ts(43,31): error TS2339: Property 'source' does not exist on type 'PluginMetadata'.
lerna ERR! [compile] src/browser/plugin/che-plugin-frontend-service.ts(44,34): error TS2339: Property 'source' does not exist on type 'PluginMetadata'.
lerna ERR! [compile] src/browser/plugin/che-plugin-frontend-service.ts(46,38): error TS2339: Property 'source' does not exist on type 'PluginMetadata'.
lerna ERR! [compile] src/browser/plugin/che-plugin-frontend-service.ts(46,64): error TS2339: Property 'source' does not exist on type 'PluginMetadata'.
lerna ERR! [compile] src/browser/plugin/che-plugin-frontend-service.ts(46,90): error TS2339: Property 'source' does not exist on type 'PluginMetadata'.
lerna ERR! [compile] src/browser/plugin/che-plugin-frontend-service.ts(50,38): error TS2339: Property 'source' does not exist on type 'PluginMetadata'.
lerna ERR! [compile] error Command failed with exit code 2.
```

**Comment by benoitf** [comment]
@dmytro-ndp it should be fixed, no ? 
Happy path test was successful on https://github.com/eclipse/che-theia/pull/451

**Comment by dmytro-ndp** [comment]
let me check it again

**Comment by dmytro-ndp** [comment]
Build of master has finished successfully.
